File: 1137-height-checker/height-checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> sort_heights = heights;
        int times = 0;
        sort(sort_heights.begin(), sort_heights.end());

        for (int i = 0; i < heights.size(); i++) {
            if (heights[i] != sort_heights[i]) {
                times++;
            }
        }
        return times;
    }
};

File: 1051-height-checker/height-checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        heights_max = max(heights)
        heights_min = min(heights)
        index = 0
        times = 0
        count_list = [0] * (heights_max - heights_min + 1)
        for i in range(len(heights)):
            count_list[heights[i] - heights_min ]+=1
        for i in range(len(count_list)):
            while count_list[i] > 0:
                if heights[index] != i + heights_min:
                    times += 1
                count_list[i] -= 1
                index +=1 
        return times
        